src/index.jsx

```diff
@@ -83,12 +85,27 @@ const App = () => {
     },
   );
 
+  // Global toast context to open a toast in
+  // any component of the MFE.
+  const toastContext = useToastContext();
```

Comment:
Nit: `useToastContext` sounds like it's doing `React.useContext(ToastContext)` but this is actually _creating_ the context. I would recommend slightly refactoring it like this: https://github.com/open-craft/frontend-app-course-authoring/pull/39/files. (I just made a PR because it's hard to explain :p )

Reply:
Thanks for the PR. Looks good!